### DISCUSSION TITLE
en/vctk/vits model speakers don't match VCTK dataset?

### DISCUSSION THREAD

**Discussion by mirkonasato** [coqui-ai/TTS#2283] [General Q&A]
Hi all,

I'm playing around with the `recipes/vctk/vits/train_vits.py` example, but I noticed that the voices in the `*.flac` files in the [downloaded](https://github.com/coqui-ai/TTS/blob/v0.10.2/recipes/vctk/download_vctk.sh#L6) VCTK dataset don't seem to match those generated by the pre-trained model auto-saved in `.local/share/tts/tts_models--en--vctk--vits`.

For example, speaker _p227_ sounds like a male voice in the dataset recordings, but like a very different female voice when generating speech with `synthesize.py --model_name tts_models/en/vctk/vits --speaker_idx p227`.

Just wondering why that is? I guess this has implications if one wants to fine tune an existing voice.

Thanks

**Comment by erogol** [comment]
This is voiced before and answered before. Please search the discussions

**Comment by mirkonasato** [comment]
Thanks. I did search the discussions before asking, but couldn't find the answer here. I have now searched the issues as well and found #1601.
